docs/src/pages/components/use-media-query/use-media-query.md

```diff
@@ -121,7 +121,7 @@ function handleRender(req, res) {
   const ssrMatchMedia = query => ({
     matches: mediaQuery.match(query, {
       // The estimated CSS width of the browser.
-      width: deviceType === 'mobile' ? 0 : 1024,
+      width: deviceType === 'mobile' ? '0px' : '1024px',
```

Comment:
Wasn't there a meme about JS devs preaching strict equality `===` but using `if (!value)`? That applies here: https://github.com/ericf/css-mediaquery/blob/34ba6343938e72b270205447b297611c40140a4f/index.js#L40

So `1024` would work but the numeric literal `0` doesn't since it's falsy.